### DISCUSSION TITLE
Problem with the address count used.

### DISCUSSION THREAD

**Discussion by jacquesbusnardo** [netbox-community/netbox#8525] [Q&A]
Hello,
I believe it is a BUG. 

When I mark a prefix as "Mark utilized" (192.0.2.128/25) it doesn't count with the correct usage percent(192.0.2.0/24).

![image](https://user-images.githubusercontent.com/189808/152224435-8f4b0421-9b60-4e9d-888b-9b87c578db57.png)

Is it possible to show as 100% used?

**Comment by jacquesbusnardo** [comment]
I found that if I change the prefix **status** to **Container** instead of **Active** it has the desired behavior.

**Reply by candlerb** [reply]
Yes. "Container" prefixes count all the address space in child prefixes as 100% utilized, and do not count individual IP addresses.
